container.go

```diff
@@ -860,6 +860,31 @@ func (c *Client) PutContainerArchive(id string, opts PutContainerArchiveOptions)
 	})
 }
 
+// GetContainerArchiveOptions is the set of options that can be used when
+// downloading resources from a container.
+//
+// See https://goo.gl/KnZJDX for more details.
+type GetContainerArchiveOptions struct {
+	OutputStream io.Writer `json:"-" qs:"-"`
+	Path         string    `qs:"path"`
```

Comment:
Yes it is mandatory. I agree its safer, the reason i left it inside the opts is to be consistent with the complement PutContainerArchive func. so i'll change them both to match.